docs/readmes/lte/deploy_install.md

```diff
@@ -1,46 +1,44 @@
 ---
 id: deploy_install
-title: Install AGW on Debian Stretch.
+title: Install AGW
 hide_title: true
 ---
 
-For Installing AGW on Ubuntu refer to : [_deploy_on_ubuntu_](https://magma.github.io/magma/docs/lte/deploy_install_ubuntu)
-# Install Access Gateway on Debian Stretch (Bare Metal)
-
-### AGW deployment on Debian distribution is DEPRECATED! Release 1.5 is last release supporting debian based AGW.
+# Install Access Gateway on Ubuntu (Bare Metal)
```

Comment:
From previous comment: add a big note here somewhere that Debian installs are deprecated + link to last doc+version with debian support